docs/lab/02/02.md

```diff
@@ -644,6 +644,64 @@ If the MCU provides timers, Embassy-rs will use them to suspend the software ins
 
 :::
 
+## Build and flash
+
+### Build
+
+To build a program for the Raspberry Pi Pico, run the following command (in the project root folder):
+
+`cargo build --bin <project_name>`
+
+where `<project_name>` is to be replaced with the name of your project.
+
+This will create the binary file inside `/target/thumbv6m-none-eabi/debug/`.
```

Comment:
```suggestion
This will create the binary file inside `target/thumbv6m-none-eabi/debug/`.

:::note

If the crate is part of a workspace (as it is the lab template), the `target` folder is located in the workspace's root instead of the crate's root.

:::
```